### PR TITLE
Add fix for tangram ES6 problem

### PR DESCRIPTION
**Pull request by ivanmalagon**
This PR fixes a problem with the Uglify process in Webpack and `tangram.cartodb` dependency.

`tangram.cartodb` is written in ES6 and needs to be transpiled to ES5 in order to Uglify to work correctly.

### CODE REVIEW

**Comment by ivanmalagon** [comment]
Having a problem related with webpackJsonp. 😞

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -2977,39 +2972,7 @@
     "tangram.cartodb": {
       "version": "0.4.6",
       "from": "cartodb/tangram.cartodb#master",
-      "resolved": "git://github.com/cartodb/tangram.cartodb.git#9797d1390b798c04871f44ad51887e56041cb682",
```
Comment (rjimenezda) [inline npm-shrinkwrap.json]: These changes cause the webpackJsonP error, I don't know why exactly, but revert them if you want the tests to run.

Comment (rjimenezda) [inline npm-shrinkwrap.json]: Update: reverting this breaks stuff on embed 🙃 🔫

**Comment by nobuti** [comment]
Can close this? or is a different problem than the one fixed this morning?

**Comment by nobuti** [comment]
Nothing to merge. Closing.

**Pull request closed without merging**